From: Bram Moolenaar <Bram@vim.org>
Date: Thu, 7 Feb 2019 21:27:14 +0100
Subject: patch 8.1.0878: test for has('bsd') fails on some BSD systems

Problem:    Test for has('bsd') fails on some BSD systems.
Solution:   Adjust the uname match. (James McCoy, closes #3909)
---
 src/testdir/test_functions.vim | 2 ++
 src/version.c                  | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/src/testdir/test_functions.vim b/src/testdir/test_functions.vim
index b08d9aa..e75a896 100644
--- a/src/testdir/test_functions.vim
+++ b/src/testdir/test_functions.vim
@@ -1238,6 +1238,8 @@ func Test_platform_name()
     let uname = system('uname')
     call assert_equal(uname =~? 'BeOS', has('beos'))
     call assert_equal(uname =~? 'BSD\|DragonFly', has('bsd'))
+    " GNU userland on BSD kernels (e.g., GNU/kFreeBSD) don't have BSD defined
+    call assert_equal(uname =~? '\%(GNU/k\w\+\)\@<!BSD\|DragonFly', has('bsd'))
     call assert_equal(uname =~? 'HP-UX', has('hpux'))
     call assert_equal(uname =~? 'Linux', has('linux'))
     call assert_equal(uname =~? 'Darwin', has('mac'))
diff --git a/src/version.c b/src/version.c
index 0b86826..f5f9439 100644
--- a/src/version.c
+++ b/src/version.c
@@ -791,6 +791,8 @@ static char *(features[]) =
 
 static int included_patches[] =
 {   /* Add new patch number below this line */
+/**/
+    878,
 /**/
     875,
 /**/
